### PR TITLE
Updates Teach AI section to use new strings, new styles

### PR DESCRIPTION
**Pull request by hannahbergam**
@kelbyhawn I know you might want to refactor this, but this was the quickest way to utilize the .new styles without refactoring the pegasus design system stylesheet. Currently, any '.new' tag has to be nested within an action-block, which isn't the case for this page. So I took the styles and fixed the width and got it looking correct on this page in this new context. 

Additionally, I know the figma has the image compensating for the much bigger height of the collective 'new' tag and the additional body lines. However, this is much harder to fix without messing with some very 'high up' margin stylings that dictate the max width of every section on the page. Considering we're fine with the 'text side' being bigger than the 'image side' higher up on the page, I elected to leave it.

Aside from that, this is just new strings and a new link url!

Before:
Screenshot 2024-04-15 at 5 48 13 PM

After:
Screenshot 2024-04-15 at 6 25 58 PM

## Warning!!

The [AP CSP Create Performance Task](https://apcentral.collegeboard.org/courses/ap-computer-science-principles/exam) is in progress. The most critical dates are from April 3 - April 30, 2024. Please consider any risk introduced by this PR that could affect our students taking AP CSP. Code.org students taking AP CSP primarily use App Lab for their Create Task, however a small percent use Game Lab. Carefully consider whether your change has any risk of alterering, changing, or breaking anything in these two labs. Even small changes, such as a different button color, are considered significant during this time period. Reach out to the Student Learning team or Curriculum team for more details.

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1547

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/ai/index.haml [hunk 1]
```diff
@@ -162,12 +162,14 @@ theme: responsive_full_width
     %figure.col-45{style: "margin-bottom: 1rem"}
       %img{src: "/images/ai/pl/101/TeachAI.png", alt: "", style: "width: 100%"}
     .text-wrapper.col-50{style: "margin-bottom: 1rem"}
+      %p.new
+        =hoc_s(:new)
       %h2
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/index.haml]:
```suggestion
      %h2.heading-lg
```

File: pegasus/sites.v3/code.org/public/ai/index.haml [hunk 2]
```diff
@@ -162,12 +162,14 @@ theme: responsive_full_width
...
-        =hoc_s(:teach_ai_title)
+        =hoc_s(:ai_promote_literacy)
       %p
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/index.haml]:
```suggestion
      %p.body-three
```

**Review by kelbyhawn — COMMENTED** [review]
This solution works fine for now! 

I left two suggestions that should help with the spacing issues so it'll look like this:
Screenshot 2024-04-16 at 9 05 01 AM

**Comment by hannahbergam** [comment]
> This solution works fine for now!
> 
> I left two suggestions that should help with the spacing issues so it'll look like this: Screenshot 2024-04-16 at 9 05 01 AM

Love it- just making those things a smidge smaller! Done:)

**Review by kelbyhawn — APPROVED** [review]

**Pull request merged**